client/src/selectors/index.js

```diff
@@ -54,3 +54,12 @@ export const paginatedNoticesSelector = createSelector(
 export const noticesNumPagesSelector = createSelector(dateSortedNoticesSelector, (dateSorted) =>
   Math.ceil(dateSorted.length / paginationChunkSize)
 )
+
+
+export const PATH_MAP_OPTIONS = ['mapOptions']
+
+export const mapOptionsSelector = (state: State): MapOptions => get(state, PATH_MAP_OPTIONS)
```

Comment:
I would refrain from using get where it's not needed (as well as defining paths in this way) - the reason being you no longer get static checking

I'd prefer to have that - along with, often, a more concise/expressive piece of code (I prefer `state.substate.something` to `get(state, PATH_CONSTANT_I_HAVE_TO_LOOK_FOR)` - maybe expect when the path would be too long, but then again, we want to refrain from excessive nesting)

The only other benefit of this approach I can see is when a piece of state moves elsewhere - but then I'd rather again be guided by flow errors instead of hoping that the constant was used everywhere

Reply:
This is good point, but what about paths in actions? We lose the static type check there don't we?